packages/core/deploy/002_deploy_timer.js

```diff
@@ -0,0 +1,20 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts, network } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const { live } = network;
+
+  if (live === undefined) throw new Error("Network has no live parameter");
+
+  // If live === false, don't deploy a timer.
+  if (live === false) {
+    await deploy("Timer", {
+      from: deployer,
+      args: [],
+      log: true,
```

Comment:
equally, seeing all of them have `log:true`, is there a way to set them all to default to `log:true` without defining it over and over again?

Reply:
It defaults to false, I don't see any info about how to make it default to true :/